source/elements/oneTBB/source/containers/concurrent_bounded_queue_cls.rst

```diff
@@ -55,6 +55,17 @@ Class Template Synopsis
 
                 ~concurrent_bounded_queue();
 
+                concurrent_bounded_queue& operator=( const concurrent_bounded_queue& other );
+                concurrent_bounded_queue& operator=( concurrent_bounded_queue&& other );
+                concurrent_bounded_queue& operator=( std::initializer_list<value_type> init );
```

Comment:
IMHO, the constructor from the `initializer_list` should also be added

Reply:
added